membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -146,4 +149,34 @@ class AttributeController(attributesFromZuora: AttributesFromZuora, commonAction
     }
   }
 
+  /**
+   * Allow all validated guardian.co.uk/theguardian.com email addresses access to the digipack
+   */
+  private def maybeAllowAccessToDigipackForGuardianEmployees(
+    maybeUser: Option[User],
+    maybeAttributes: Option[Attributes],
+    identityId: String,
+  ): Option[Attributes] = {
+
+    val allowDigiPackAccessToStaff =
+      (for {
+        user <- maybeUser
+        email = user.primaryEmailAddress
+        userHasValidatedEmail <- user.statusFields.userEmailValidated
+        emailDomain <- Try(email.split("@")(1)).toOption
```

Comment:
Would this work (/ be nicer)?
```suggestion
        emailDomain <- email.split("@").lastOption
```

Reply:
https://stackoverflow.com/questions/57959544/one-liner-to-extract-domain-from-email-address